message.go

```diff
@@ -180,6 +180,9 @@ type Message struct {
 	// in this field will not contain further ReplyTo fields even
 	// if it is itself a reply.
 	PinnedMessage *Message `json:"pinned_message"`
+
+	// Inline keyboard attached to the message.
+	ReplyMarkup InlineKeyboardMarkup `json:"reply_markup"`
```

Comment:
It's used as pointer in other places in this library
But since it's just a struct with one slice inside using it as value is better imo